index.js

```diff
@@ -1,43 +1,45 @@
 import fp from 'fastify-plugin'
-import tags from 'common-tags'
+import { html } from '@gurgunday/html'
+import { includeFile } from '@gurgunday/html/includeFile.js'
```

Comment:
Why isn't this exposed from the main export file?

Reply:
Because `html` works in the browsers too and I'd really rather not maintain separate distributions

`includeFile` would make the main export crash as it imports from "node:fs"